gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -266,7 +270,7 @@ public void orchestrate(Spec spec, Properties jobProps, long triggerTimestampMil
           return;
         }
 
-        String flowExecutionId = flowMetadata.get(TimingEvent.FlowEventConstants.FLOW_EXECUTION_ID_FIELD);
+        String flowExecutionId = TimingEventUtils.getFlowExecutionIdFromFlowMetadata(flowMetadata);
```

Comment:
What's the motivation behind changing this function here? From my point of view we should be enforcing the existence of a flow execution ID here as a dag must have an associated flow execution ID, otherwise we should block progression as the dag is likely corrupt

Reply:
Ah I see, I changed this to abstract away the use of the key. I could update the function to not provide a default if one doesn't exist, always expecting existence of flow execution id or revert this. What do you think?

Reply:
Sure in that case